mediacenter/utils.py

```diff
@@ -0,0 +1,14 @@
+import mimetypes
+
+
+def guess_kind(filepath):
+    """Return a document kind (image, audio...) guessed from a filename. If
+    no kind can be guessed, returns None."""
+    lookups = ['image', 'video', 'audio', 'text', 'pdf']
+    if filepath:
+        kind, encoding = mimetypes.guess_type(filepath)
+        if not kind:
+            return
+        for lookup in lookups:
+            if lookup in kind:
+                return lookup
```

Comment:
Right! But we are doing our guess_type at save time, which doesn't means we are on the upload file process. The fact is that later we'll have command line for batch import of documents, which then will not come from a Form. Do that makes sense? :)